src/components/graphs/AnomalyAnnualCycleGraph.js

```diff
@@ -25,44 +25,60 @@ import AnnualCycleGraph from './AnnualCycleGraph';
 
 export default function AnomalyAnnualCycleGraph(props) {
   
-  function getPresentMetadatas(includeFuture = false) {
-    // returns an array of all metadata objects that conform 
-    // to the user-selected parameters and contain data that 
-    // includes the current year (or future years, if includeFuture is true).
-    const currentYear = new Date().getFullYear();
-    
+  function getDateRangeMetadatas(start = undefined, end = undefined) {
+    //returns metadata for all datasets whose start year is greater 
+    //than start, and whose end year is less than end.
+    //the interval is exclusive. optionally leave either endpoint undefined.
```

Comment:
Why is the interval exclusive?

Reply:
It was convenient for the [task at hand](https://github.com/pacificclimate/climate-explorer-frontend/issues/175), which was finding a "historical" baseline, ie, one that does _not_ contain the present year.

Reply:
If it's exclusive on both ends, then a date that exactly a boundary won't get found. Does that matter?

Reply:
Nevermind, clearly not. I'm content to leave this as is.

Reply:
At present, it is the desired behaviour both times the function is used:

1. To find the set of all "historical" climatologies, where "historical" is implicitly defined as "all climatologies whose end date is strictly earlier than the present year"
2) having selected a baseline "historical" climatology, to find the set of all "future" climatologies relative to it, where "future" is implicitly defined as "all climatologies whose start year is strictly later than the start year of the base climatology"

I wanted a baseline / future selection process that would reasonably handle the case when the desired baseline climatology (1971-2000) wasn't present. It's not a very complicated algorithm, and if we get overlapping climatologies we may want to revisit these definitions of "historical" and "future."